analytical/templatetags/piwik.py

```diff
@@ -41,7 +41,19 @@
 
 VARIABLE_CODE = '_paq.push(["setCustomVariable", %(index)s, "%(name)s", "%(value)s", "%(scope)s"]);'  # noqa
 IDENTITY_CODE = '_paq.push(["setUserId", "%(userid)s"]);'
-DISABLE_COOKIES_CODE = '_paq.push([\'disableCookies\']);'
+DISABLE_COOKIES_CODE = "_paq.push(['disableCookies']);"
+ASK_FOR_CONSENT_CODE = "_paq.push(['requireConsent']);"
+FORGET_CONSENT_CODE = """
+document.getElementById("piwik_deny_consent").addEventListener("click",
+() => {
```

Comment:
No real reason not to support poor souls forced to use IE, right? Replace with old-school `function () { ... }`?

Reply:
Changed in c3065fc0c96e762f064176f16554af440055cadc